example/centos/centos-9-x86_64-oci.yaml

```diff
@@ -0,0 +1,174 @@
+otk.version: "1"
+
+otk.define:
+  filesystem:
+    modifications:
```

Comment:
(nitpick) for consistency with the other a `# empty` comment here would be good I think to indicate that we left this empty on purpose.

Reply:
Done.